configure: on Linux, the BSD functions require linking against libbsd

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN libedit-20130712-3.1.orig/configure.ac libedit-20130712-3.1/configure.ac
--- libedit-20130712-3.1.orig/configure.ac	2013-07-12 20:39:22.000000000 +0200
+++ libedit-20130712-3.1/configure.ac	2013-07-28 23:35:35.012676469 +0200
@@ -108,6 +108,10 @@
 #AC_FUNC_REALLOC
 AC_TYPE_SIGNAL
 AC_FUNC_STAT
+
+# Search for libbsd for *vis functions
+AC_CHECK_LIB([bsd], [vis])
+
 AC_CHECK_FUNCS([endpwent isascii memchr memset re_comp regcomp strcasecmp strchr strcspn strdup strerror strrchr strstr strtol issetugid wcsdup strlcpy strlcat fgetln vis strvis strunvis __secure_getenv secure_getenv])
 
 # strlcpy
